pkg/volume/csi/nodeinfomanager/nodeinfomanager_test.go

```diff
@@ -51,8 +50,8 @@ type testcase struct {
 	inputNodeID         string
 	inputTopology       map[string]string
 	inputVolumeLimit    int64
+	expectedCSINode     *storage.CSINode
 	expectedNodeIDMap   map[string]string
```

Comment:
Can we also replace expectedNodeIDMap, expectedLabels, expectedVolumeLimit with one expectedNode object?